spanishdict/django-nested-inlines
# Bug 177908778 add permissions django 3

@seankarson 

### What
Content reported they are unable to add entries.

When I tried locally this is the error I see "has_add_permission() takes 2 positional arguments but 3 were given"

Per Django 3 release notes "The shim to allow InlineModelAdmin.has_add_permission() to be defined without an obj argument will be removed." However it looks like ModelAdmin does not take in obj, so reverting the `has_add_permissions` updates and replacing with a comment on why we have to keep it in

### How to test
Take the commit from the second commit and update in Hegemone's requirements.txt and rebuild. Try adding a dictionary entry, should be able to get to page. (will make a commit on Hegemone whenever this is approved and merged)

cc: @iraquint
